### PR TITLE
MMT-3410: Adds UMM-T draft forms

### PR DESCRIPTION
**Pull request by macrouch**

### CODE REVIEW

**Review by trevorlang — APPROVED** [review]

File: static/src/js/components/CustomRadioWidget/CustomRadioWidget.jsx [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+  id,
+  label,
+  onChange,
+  registry,
+  required,
+  schema,
+  uiSchema,
+  value
+}) => {
+  console.log('🚀 ~ file: CustomRadioWidget.jsx:40 ~ id:', id)
```
Comment (tle1989) [inline static/src/js/components/CustomRadioWidget/CustomRadioWidget.jsx:40]: Print statement, remove?

**Review by cgokey — APPROVED** [review]

File: static/src/js/components/FormNavigation/FormNavigation.jsx [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+        </Button>
+      </div>
+
+      <ListGroup className="form-navigation__sections p-3 bg-light">
+        <For each={formSections}>
+          {
+            (section, index) => {
+              const { displayName } = section
+
+              // TODO MMT-####
```
Comment (dmistry1) [inline static/src/js/components/FormNavigation/FormNavigation.jsx:140]: Is this something we had in the prototype?

Comment (macrouch) [reply static/src/js/components/FormNavigation/FormNavigation.jsx:140]: I was waiting on a ticket Trevor was going to write lol, need to get a real ticket number in each place that shows up

File: static/src/js/components/CustomMultiSelectWidget/CustomMultiSelectWidget.jsx [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+  }
+
+  const handleBlur = () => {
+    setFocusField(null)
+
+    onBlur(id)
+  }
+
+  // If the value already has data, this will store it as an object for react-select
+  const existingValues = value.filter(Boolean).map((currentValue) => ({
```
[the remark this answers is not in the document]
Comment (macrouch) [reply static/src/js/components/CustomMultiSelectWidget/CustomMultiSelectWidget.jsx:91]: `.filter(Boolean)` removes any empty values from an array, so if `value` is `[undefined]` that filter will return `[]`

File: static/src/js/components/NavigationItemError/NavigationItemError.scss [hunk 4]
```diff
@@ -0,0 +1,32 @@
+.navigation-item-error {
+  &__item {
+    // width: 330px;
```
Comment (dmistry1) [inline static/src/js/components/NavigationItemError/NavigationItemError.scss:3]: These can be removed

File: static/src/js/components/CustomTextWidget/CustomTextWidget.jsx [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+  const { formContext } = registry
+  const {
+    focusField,
+    setFocusField
+  } = formContext
+
+  const { maxLength, description } = schema
+
+  const fieldType = uiSchema['ui:type']
+  const headerClassName = uiSchema['ui:header-classname']
```
[the remark this answers is not in the document]
Comment (macrouch) [reply static/src/js/components/CustomTextWidget/CustomTextWidget.jsx:56]: Take a note of this for your work on services, I don't want to mess with this before our demo

**Review by dmistry1 — APPROVED** [review]

**Review by htranho — APPROVED** [review]

**Pull request merged**